GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
+        if let colorValue = keyColorPair.1?.rgbColor?.hexValue {
+          addColorWithName(colorName, value: colorValue)
+        }
+      }
+    }
+  }
+  
+  public func addColorWithName(name: String, value: String) {
+    addColorWithName(name, value: ColorsTextFileParser.parse(value))
```

Comment:
No I think we should remove them, that does not make sense that the `CLRFileParser` has a method to add a color by passing its value as an hex `String` (whereas it makes sense for the `ColorsTextFileParser` because that's the format which is expected in the text file).

We shouldn't clobber the API of the class _just_ because it's easier to test.